City-of-Helsinki/drupal-helfi-sote
# UHF-7782: Hesteri chatbot config

## What has been done
- added block visibility (or rather "invisibility") config for the new Hesteri chatbot as described in the ticket https://helsinkisolutionoffice.atlassian.net/browse/UHF-7782
- removed the SUUNTE Genesys chat as it is replaced by the new chatbot

## How to install
1. set up SOTE instance (try to get the production database if possible as it is easier to verify the block visibility in required paths)
2. checkout this branch: `git checkout UHF-7782_sote_chatbot_config`
3. import config and clear cache: `make drush-cim drush-cr`
4. set the config values for the "IBM Chat App" block from the comment https://helsinkisolutionoffice.atlassian.net/browse/UHF-7766?focusedCommentId=75469

## How to test
1. view the paths defined in the ticket and verify that the chatbot is visible
2. view (at least some) paths outside of that scope and verify that the chatbot is not visible
